Dockerfile

```
@@ -6,8 +6,9 @@ RUN mkdir -p /usr/src/build && \
     cd /usr/src/build && \
     pip3 install --upgrade pip setuptools wheel cython && \
     pip3 install -r /usr/src/app/requirements.txt && \
+    chmod +x /usr/src/app/script/install-coap-client.sh && \
     /usr/src/app/script/install-coap-client.sh && \
-    python /usr/src/app/setup.py install 
+    python /usr/src/app/setup.py install
```

I'd install with `pip` in development mode.
```suggestion
    pip3 install -e /usr/src/app/
```